test/test-server/networking_handlers.go

```diff
@@ -1371,6 +1373,56 @@ func handleNetworkingNetworkLinkServiceCreate(t *testing.T) http.HandlerFunc {
 	}
 }
 
+func handleNetworkingNetworkLinkServiceUpdate(t *testing.T, id string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch id {
+		case "nls-invalid":
+			w.WriteHeader(http.StatusNotFound)
+			err := writeErrorJson(w, "The network link service nls-invalid was not found.")
```

Comment:
I tested the API and the error message is "NetworkLinkService is not found."

I think we would want to have the actual error messages from the API in our integration tests since contract tests will be based on these integration tests (once we start using contract tests in the CLI). @brianstrauch could you please confirm?

@ay-zhang Can you double check the error messages for this update command and the other network link commands as well?

Reply:
I think it's a great idea to have them match. Can we update that backend error message to be match this one though? (This one is more helpful)

Reply:
I believe there's more work involved for updating the backend error message and I do not want to expand the scope for this project. We can certainly mark it as an action item but let's also check with @ejagade.